plothist/variable_registry.py

```diff
@@ -266,7 +266,7 @@ def update_variable_registry_ranges(
     Raises
     ------
     RuntimeError
-        If the variable does not have a bins or range property in the registry.
+        If the variable does not have a name, bins and/or range property in the registry.
```

Comment:
I do not understand the and/or. The error is thrown if the variable is missing a name, bin or range field.